chemistry/amber/amberformat.py

```diff
@@ -788,6 +787,27 @@ def set_version(self):
 
     #===================================================
 
+    def convert_to_vmd_compat(self):
```

Comment:
`AmberFormat` is a base class to the actual topology file objects.  `AmberParm` inherits from `AmberFormat` and both `ChamberParm` and `AmoebaParm` inherit from `AmberParm`.  This method is specific to chamber topology files (and will not in fact work with any other flavor).  So can you move this function to the `ChamberParm` class in `chemistry/amber/_chamberparm.py`?